tests/test_data_integrity.py

```diff
@@ -143,7 +143,6 @@ def validator(data):
 
         interfaces.library.validate(validator, success_by_default=True)
 
-    @missing_feature(library="nodejs", reason="not implemented yet")
```

Comment:
instead of removing the skip, please use a conditional skip. This will allow us to run the test suite on older versions of the library without having failures everywhere. Here is an example (github won't let me do a suggestion):
```python
@missing_feature(context.library < "nodejs@5.47.0", reason="not implemented yet")
```

Reply:
I thought the YAML would be used for this? Why repeat this in multiple places?

Reply:
Because of specific and well thought-out reasons, the manifest can control the skips for class-level, but not for method-level stuff. So if you want to skip a specific method inside a test class, you can't use the manifest, you have to add a decorator.
It's not a repetition, it's a "detailing"